staging/src/k8s.io/apimachinery/pkg/apimachinery/registered/registered.go

```diff
@@ -222,6 +222,8 @@ func (m *APIRegistrationManager) GroupOrDie(group string) *apimachinery.GroupMet
 	groupMeta, found := m.groupMetaMap[group]
 	if !found {
 		if group == "" {
+			// If a test triggers this panic, assure that the test's package
+			// imports "k8s.io/kubernetes/pkg/api/testapi" to run its init.
```

Comment:
@deads2k It will be part of a big refactoring. I'll find time describe the ideas. Maybe in the next release.

@monopole could you instead instruct people to directly import the [install packages](https://github.com/kubernetes/kubernetes/tree/master/pkg/api/install)? That will make later refactoring easier.